CoreSchemaView/release/modules/ext/docs/CoreSchemaView/src/au/gov/asd/tac/constellation/views/schemaview/schemas.md

```diff
@@ -0,0 +1,25 @@
+# Schemas
+
+Schemas are used to help Constellation interpret the data that is stored on the graph. Each
+graph type in Constellation has a 'schema' associated with it, allowing it to understand
+certain types of data and giving it the opportunity to improve data quality and visualisation
+based on this knowledge. 
+
+When you create a new graph, you can specify the schema that the graph will use:
+-   Visual Graph
+-   Analytic Graph
+
+## Visual Graph
```

Comment:
I think I would rename the references to the graph types to the schema types (e.g. Visual Schema rather than Visual Graph) as in this case you are describing the schema, not the graph that the schema backs.